tests/test_unit/test_report_a_suspected_breach/test_views/test_views_start.py

```diff
@@ -5,7 +5,22 @@
 from django.test import RequestFactory
 from django.urls import reverse
 from report_a_suspected_breach.models import ReporterEmailVerification, Session
-from report_a_suspected_breach.views.views_start import EmailVerifyView
+from report_a_suspected_breach.views.views_start import (
+    EmailVerifyView,
+    WhatIsYourEmailAddressView,
+)
+
+
+class TestWhatIsYourEmailAddressView:
+    def test_post(self, rasb_client):
+        request_object = RequestFactory().get("/")
+        request_object.session = rasb_client.session
+        view = WhatIsYourEmailAddressView()
+        view.setup(request_object)
+        data = {"reporter_email_address": "test@123.com"}
+        response = view.post(request_object, data)
+
+        assert response.status_code == 200
```

Comment:
the issue here is that even an invalid form will return a 200, as it's just a simple template response with the form errors in the header

Reply:
It would be more reliable to check the form object itself. e.g.

```suggestion
        assert response.context["form"].is_valid()
```

Reply:
I committed the above but couldn't get it passing. I tried using the view.post and via the rasb_client.post but couldn't get the form object in either. I instead updated the test to look for the correct success url in the response, with a 302 redirect.